content/home/marketing/component-based.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Component-Based
+title: Bazat pe componente
 order: 1
 ---
 
-Build encapsulated components that manage their own state, then compose them to make complex UIs.
+Construiește componente încapsulate care își administrează propria stare, apoi compune-le pentru a crea interfețe complexe.
```

Comment:
I'm sorry but I will disagree here.
https://dexonline.ro/definitie/%C3%AEncapsulat

And "compune-le" comes from combining the pronoun "le" with the conjugated verb "compune"

Reply:
@RaulTsc what do you think?

Reply:
@gabrielmoncea I think I am crazy 🤣 don't know how I thought of that..it it obviously "apoi le compune" instead of my first try - compunele(nice catch) & for "încapsulate" I think you are right. 👍

Reply:
In this context it's about the programmer that builds and composes the UI, not React. That's why I used "compune-le" and not "le compune"